### ISSUE TITLE
Any human way to compile qt wallet for win64?

### ISSUE THREAD

**Issue by ScaMar** [PIVX-Project/PIVX#640]
### Can you reliably reproduce the issue?
create a virtual machine
install win10
install WSL with ubuntu 16.06

#### If so, please list the steps to reproduce below:

1. sudo apt update
2. sudo apt upgrade
3. sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl git
4. sudo apt install g++-mingw-w64-x86-64
5. sudo apt install unzip
6. sudo update-alternatives --config i686-w64-mingw32-g++ #POSIX will be the default one
7. cd PIVX-3.1/depends/
8. PATH=$(echo "$PATH" | sed -e 's/:\/mnt.*//g')
9. cd depends
10. make HOST=x86_64-w64-mingw32
11. cd ..
12. ./autogen.sh
13. CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/opt/local
14. make

### Expected behavior
A working executable named pivx-qt.exe

### Actual behavior
A broken executable

### Screenshots.
![image](https://user-images.githubusercontent.com/1008128/41669865-12c69ae8-74b3-11e8-859f-a4a579a8f1f0.png)

### What version of PIVX Core are you using?
compiling zip file 3.1.zip

### Machine specs: (vmware)
- OS: win10
- CPU: i7
- RAM: 2Gb
- Disk size: 60gb
- Disk Type (HD/SDD): ssd

### Any extra information that might be useful in the debugging process.
A working procedure will be appreciated instead search info from DASH and bitcoin

**Comment by Warrows** [comment]
You are almost there. I think you just have two small mistakes:

Step 10: make HOST=i686-w64-mingw32
Step 13: CONFIG_SITE=$PWD/depends/i686-w64-mingw32/share/config.site ./configure --prefix=/

Let me know if it works, and if so we'll update documentation accordingly.

**Comment by ScaMar** [comment]
@Warrows correct me if i'm wrong:
using your steps instead mine, i will build (eventually) the 32bit version. I'm trying to build the 64bit.
btw thank you for the input

**Comment by Warrows** [comment]
You are absolutely right. I did not stop to figure out why 32bit is working and 64 is not. But that's the way it is on my system. Now that I think about it, it makes no sense since we build and distribute both with gitian.

**Comment by ScaMar** [comment]
Fresh vm with ubuntu 14.04,
the steps above worked as expected.
The pivx-qt.exe works on win7 and win10.

I got an error after `make deploy` in order to create the exe package.
The executable seems to work fine.

**Comment by Warrows** [comment]
I believe `make deploy` is intended for MacOS. From wsl, you will want to use something like `make install DESTDIR=/mnt/c/workspace/pivx` to put out all binaries in a single folder.

**Comment by ScaMar** [comment]
on macos, not cross compiling, not using ccache, just native compiling with newer qt and openssl the  ```make deploy``` command created the typical dmg file.
On windows, cross compiling from linux ``` make deploy``` should create the exe package
```$ make deploy
Making all in src
make[1]: Entering directory `/home/utente/PIVX-3.1/src'
make[2]: Entering directory `/home/utente/PIVX-3.1/src'
make[2]: Leaving directory `/home/utente/PIVX-3.1/src'
make[1]: Leaving directory `/home/utente/PIVX-3.1/src'
make[1]: Entering directory `/home/utente/PIVX-3.1'
make[1]: Nothing to be done for `all-am'.
make[1]: Leaving directory `/home/utente/PIVX-3.1'
/bin/mkdir -p ./release
STRIPPROG="/usr/bin/x86_64-w64-mingw32-strip" /bin/bash /home/utente/PIVX-3.1/build-aux/install-sh -c -s ./src/pivxd.exe ./release
STRIPPROG="/usr/bin/x86_64-w64-mingw32-strip" /bin/bash /home/utente/PIVX-3.1/build-aux/install-sh -c -s ./src/qt/pivx-qt.exe ./release
STRIPPROG="/usr/bin/x86_64-w64-mingw32-strip" /bin/bash /home/utente/PIVX-3.1/build-aux/install-sh -c -s ./src/pivx-cli.exe ./release
error: could not build pivx-3.1.0-win64-setup.exe
built pivx-3.1.0-win64-setup.exe
```
In somewhere i found it may depend on missing package ```nsis```, btw there is no info about this package in doc

Also the strip isn't efficient as linux native or macos native, the exe is about 30Mb (45Mb withouth stripping debug info). On macos and linux the qt wallet is less than 15Mb.
